homeassistant/components/binary_sensor/insteon_plm.py

```diff
@@ -56,32 +61,58 @@ def should_poll(self):
     @property
     def address(self):
         """Return the address of the node."""
-        return self._address
+        return self._device.address.human
 
     @property
-    def name(self):
+    def id(self):
         """Return the name of the node."""
-        return self._name
+        return self._device.id
```

Comment:
Is this unique enough to be a `unique_id` ?

Reply:
id property is no longer used and has been removed.